android/app/src/main/res/values-da/strings.xml

```diff
@@ -771,10 +771,13 @@
 	<string name="type.amenity.parking.private">Privat parkering</string>
 	<string name="type.amenity.parking.park_and_ride">Parkering</string>
 	<string name="type.amenity.parking.underground">Underjordisk parkering</string>
+	<string name="type.amenity.parking.fee.underground">Underjordisk parkering</string>
+	<string name="type.amenity.parking.underground.private">Underjordisk parkering</string>
+	<string name="type.amenity.parking.private.underground">Underjordisk parkering</string>
 	<string name="type.amenity.parking.street_side">Parkering</string>
 	<string name="type.amenity.parking.fee.street_side">Parkering</string>
-	<string name="type.amenity.parking.lane">Parkering</string>
-	<string name="type.amenity.parking.fee.lane">Parkering</string>
+	<string name="type.amenity.parking.street_side.private">Parkering</string>
+	<string name="type.amenity.parking.private.street_side">Parkering</string>
```

Comment:
Should we clarify translations for these subtypes to make icons meaning very clear?

Reply:
I added translations. They need to be reviewed by translators